source/features/close-out-of-view-dropdowns.js

```diff
@@ -0,0 +1,41 @@
+import select from 'select-dom';
+import delegate from 'delegate';
+
+const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
+	if (intersectionRatio === 0) {
+		if (target.classList.contains('dropdown-menu') && select.exists('.dropdown-details[open]')) {
+      // Close <details> dropdowns if it's still open
```

Comment:
@sindresorhus why isn't this indentation detected? eslint bug or xo? Eslint seems to detect it with `indent: "2"`

Reply:
@bfred-it XO is not yet using ESLint 4 (It will soon). Just tried it, and it will be correctly detected then.